src/components/Homepage/BodySection/CallToAction.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Icon from '@ably/ui/core/Icon';
+import Link from '../../Link';
+import { CallToActionProps } from '../HomepagePageContent';
+
+export const CallToAction = ({ callToAction }: { callToAction: CallToActionProps }) => {
+  return (
+    <div className="flex justify-center mt-40">
+      <Link to={callToAction.href} className="ui-btn font-medium">
```

Comment:
I think we could do with some destructuring here! we don't often access object properties directly when passing props...